client/post-editor/editor-drawer/taxonomies.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { connect } from 'react-redux';
+import map from 'lodash/map';
+
+/**
+ * Internal dependencies
+ */
+import QueryTaxonomies from 'components/data/query-taxonomies';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getEditorPostId } from 'state/ui/editor/selectors';
+import { getEditedPostValue } from 'state/posts/selectors';
+import { getPostTypeTaxonomies } from 'state/post-types/taxonomies/selectors';
+import Accordion from 'components/accordion';
+import Gridicon from 'components/gridicon';
+
+function EditorDrawerTaxonomies( { siteId, postType, taxonomies } ) {
+	return (
+		<div className="editor-drawer__taxonomies">
+			{ siteId && postType && (
+				<QueryTaxonomies { ...{ siteId, postType } } />
+			) }
+			{ map( taxonomies, ( taxonomy ) => {
+				if ( 'post_format' === taxonomy.name ) {
+					// Post format has its own dedicated accordion
+					return;
+				}
+
+				return (
+					<Accordion
+						key={ taxonomy.name }
+						title={ taxonomy.label }
+						icon={ <Gridicon icon="tag" /> }>
```

Comment:
>Need some fancier gridicons here 😆 

Yeah, the repetition of the tag icon in the [example screenshot](https://cloud.githubusercontent.com/assets/1779930/15125886/38568e38-15fd-11e6-85d2-0948344479f5.png) leaves some to be desired. Perhaps we can hard-code icons for the known post types taxonomies? Or choose a different icon for hierarchical and non-hierarchical taxonomies, since at least in this case and for posts, there's one of each (Tags and Types/Categories).

Reference: https://wpcalypso.wordpress.com/devdocs/design/gridicons

_("folder" for hierarchical types?)_

/cc @hugobaeta